BookView에 AddSnaphotListner 추가 및 자기 글 보이도록 변경

hekang42 opened the pull request:
## 제목
 BookView에 AddSnaphotListner 추가. 
## 내용
 실시간 업데이트가 됨.
 자기 글이 보임.


## 주의사항
 나머지 리스트 뷰들에도 적용해야함.

hekang42 committed:
Feature Email Verification in sign in & sign up  * closed #197

hekang42 committed:
Merge remote-tracking branch 'refs/remotes/origin/main'

hekang42 committed:
Merge remote-tracking branch 'refs/remotes/origin/main'

hekang42 committed:
Fix : 자신의 글이 보이도록 수정. #199

hekang42 committed:
Book View에 AddSnapshotListner 적용. 실시간으로 업데이트 됨.
AddBook View에서 이미지를 저장하고 파일을 업로드해서 실시간 업데이트에 이미지가 보이도록 딜레이를 줄임.

sushin-42 reviewed (dismissed):
GJ :) db에서 불러오는 작업을 하나로 통합 할 날이 곧 오겠네요

hekang42 committed:
add filtering Block & Report in books

jinbekim reviewed (dismissed):
gj

hekang42 committed:
Fix Book View to getImage

hekang42 committed:
Fix NeighborGridView to using AddSnapShotListner

hekang42 committed:
프로필 뷰에서 북마크 버튼 보이도록 수정.
관련 코드들 삭제.
BookView에서 내 책 눌렀을때 나오는 버튼 프로필뷰와 동일하게 보이도록 수정.

sushin-42 reviewed (approved):
lgtm

jinbekim reviewed (approved):
lgtm
